test/python/test_examples.py

```diff
@@ -44,15 +43,15 @@ def test_all_examples(self):
                 example_path = os.path.join(examples_dir, example)
                 cmd = [sys.executable, example_path]
                 run_example = subprocess.Popen(cmd, stdout=subprocess.PIPE,
-                                               stderr=subprocess.PIPE)
+                                               stderr=subprocess.PIPE,
+                                               env={**os.environ, "PYTHONIOENCODING": "utf8"})
                 stdout, stderr = run_example.communicate()
                 error_string = "Running example %s failed with return code %s\n" % (
                     example, run_example.returncode)
                 error_string += "stdout:%s\nstderr: %s" % (
-                    stdout.decode('utf8'), stderr.decode('utf8'))
+                    stdout, stderr)
                 self.assertEqual(run_example.returncode, 0, error_string)
 
-    @unittest.skipIf(os.name == 'nt', 'Skip on windows until #2616 is fixed')
     @unittest.skipIf(sys.platform == 'darwin' and sys.version_info[1] == 8,
```

Comment:
Don't we need to set the encoding in this test too?

Reply:
Indeed. Fixed in 449a3d5